src/v2i-hub/MessageLoggerPlugin/src/MessageLoggerPlugin.cpp

```diff
@@ -317,7 +317,7 @@ void MessageLoggerPlugin::HandleBasicSafetyMessage(BsmMessage &msg,
 
 
 	if(bsm->partII != NULL) {
-		if (bsm->partII[0].list.count >= partII_Value_PR_SpecialVehicleExtensions ) {
+		if (bsm->partII[0].list.count >= PartIIcontent_126P0__partII_Value_PR_SpecialVehicleExtensions ) {
```

Comment:
This is all good but I am worried if this breaks other plugins. I dont think other plugins are using _126P0_ as the variable names.

Reply:
I fixed this every where it occured and cause compilation errors. It seemed easier and better to fix the code that used the enum rather that change the autogenerated name of the enum from the asn1 compiler.